src/main/java/uk/gov/ons/census/casesvc/service/ReceiptProcessor.java

```diff
@@ -58,12 +58,16 @@ public void processReceipt(ResponseManagementEvent receiptEvent) {
     }
 
     UacQidLink uacQidLink = uacQidLinkOpt.get();
-    Case caze = uacQidLink.getCaze();
+    uacQidLink.setActive(false);
```

Comment:
This could be called receiptedCase instead of "caze"

Reply:
good ideas @rjweeks but it's caze everywhere else so reluctant to change it